cmd/apiserver/app/server/run_server.go

```diff
@@ -106,6 +108,11 @@ func runEtcdServer(opts *ServiceCatalogServerOptions, stopCh <-chan struct{}) er
 	// run in addition the checkers being installed here.
 	server.GenericAPIServer.AddHealthzChecks(etcdChecker)
 
+	// Register handler to return Service Catalog version (version format is "v0.1.35-14+e21502afa7d08e")
+	server.GenericAPIServer.Handler.NonGoRestfulMux.HandleFunc("/service-catalog-version", func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, "%q", html.EscapeString(pkg.VERSION))
```

Comment:
Do we want to match the basic structure what /version does? 

    {
      "major": "1",
      "minor": "11",
      "gitVersion": "v1.11.3",
      "gitCommit": "a4529464e4629c21224b3d52edfe0ea91b072862",
      "gitTreeState": "clean",
      "buildDate": "2018-09-09T17:53:03Z",
      "goVersion": "go1.10.3",
      "compiler": "gc",
      "platform": "linux/amd64"
    }
    
e.g. 
    
    { "gitVersion": "v0.1.0" }

That way down the road, if we need to add more fields, we aren't limited by the unstructured payload.

Reply:
Absolutely, I considered it but ended up with a simple string.   I'll update it.